src/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -203,7 +203,7 @@ function validateParams(vmParams) {
 
     if (vmParams.nodeMaxMemory && vmParams.memorySize > convertToUnit(vmParams.nodeMaxMemory, units.KiB, vmParams.memorySizeUnit)) {
         validationFailed.memory = cockpit.format(
-            _("Up to $0 $1 available on the host"),
+            _("$0 $1 available on host"),
```

Comment:
What's the rationale for this? This breaks 2x38 translations.

Reply:
It was discussed in [this comment](https://github.com/cockpit-project/cockpit-machines/pull/560#issuecomment-1062915042) and 2 comments above.